src/Valcheck.js

```diff
@@ -228,6 +251,8 @@ class Valcheck {
     var error;
     if ((error = this.string(key, value, true))) { return error; }
 
+    if (CSS_COLORS.includes(value)) { return; }
```

Comment:
please use `CSS_COLORS.indexOf(value) >= 0` since we allow older versions of node